build/assets/themes/concrete_cms/scss/main.scss

```diff
@@ -51,7 +51,11 @@ div.ccm-page {
   flex-direction: column;
 
   main {
-    flex: 1;
+    flex: 1 0 0;
+
+    @-moz-document url-prefix() {
+      flex: 1;
```

Comment:
It'd be nice if there were a comment included that made it clear that this block is here to only target firefox.